3.0.0/reference.json

```diff
@@ -1453,6 +1453,34 @@
                 "default-meaning": "The geometry will not be clipped to map bounds before rendering.",
                 "doc": "Turning on clipping can help performance in the case that the boundaries of the geometry extent outside of tile extents. But clipping can result in undesirable rendering artifacts in rare cases."
             },
+            "simplify": {
+                "css": "shield-simplify",
+                "type": "float",
+                "expression": true,
+                "default-value": 0.0,
+                "default-meaning": "geometry will not be simplified.",
+                "doc": "Simplify the geometries used for shield placement by the given tolerance."
+            },
+            "simplify-algorithm": {
+                "css": "shield-simplify-algorithm",
+                "type": ["radial-distance",
+                         "zhao-saalfeld",
+                         "visvalingam-whyatt"
+                        ],
+                "expression": true,
+                "default-value": "radial-distance",
+                "default-meaning": "The geometry will be simplified using the radial distance algorithm.",
+                "doc": "Simplify the geometries used for shield placement by the given algorithm."
+            },
+            "smooth": {
+                "css": "shield-smooth",
+                "type": "float",
+                "expression": true,
+                "default-value": 0,
```

Comment:
For consistency, I guess we should have `0.0` here as a default, given that it's a `float` (or to put 0 everywhere else ;) ).